docs/source/models/concept_models/vivarium_nutrition_optimization/kids/concept_model.rst

```diff
@@ -655,6 +662,40 @@ Baseline values for :ref:`wasting treatment <intervention_wasting_tx_combined_pr
 
   :math:`E_\text{SAM}` and :math:`E_\text{MAM}` parameter values will **not** vary by scenario in this model.
 
+
+Scenario Runs for Targeted SQ-LNS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Due to logistic limitations, running 543 is likely unfeasible. Given this, there are two 
+main options for limiting the scenarios needed. First, we can limit what antenatal 
+scenarios and child scenarios are run together. Or second, we can limit what child 
+products are run simultaneously. 
+
+From our prior work on this model, we have seen that the difference between antenatal 
+products and child products is much more significant that between different 
+child product combinations. For this reason, we think it makes more sense to 
+limit the pregnancy scenarios included in all child scenario runs. 
+
+We will test this implement plan with the following steps: 
+
+1. Do a run of all child scenarios for a single pregnancy scenario and determine the optimal combination of child products. Note that yes, this could change based on the antenatal scenario and this is a limitation. 
+2. For the optimal child scenario indentified, run all pregnancy scenarios. Ensure that in all cases, MMS and BEP are both scaled up prior to any child products. If this is not true, other scenarios might be needed, such as zero coverage of antenatal products or baseline. 
+3. Assuming the assumption that MMS and BEP always scale up first holds, run the child scenarios with only the MMS+BEP pregnancy scenario. Another option would be to run the child scenarios for both MMS alone and MMS+BEP. 
```

Comment:
Hmm, I'm not sure I'm entirely following this and would love to talk through in more detail -- I may not be tracking fully, but I am concerned about how restricting scenarios could affect how we've structured the ifa/child-deleted scenarios in the emulator. I think it will probably still work as long as we guarantee that there are some scenarios/combinations of scenarios that will always be present, but I think it's something we might have to keep an eye on to make sure this works

Reply:
Basically, in the scheme that you've laid out, I think that we will also always need:
- whatever antenatal product is identified as optimal + zero child coverage
- zero antenatal + zero child
- baseline + baseline
- more thinking on which IFA scenarios would be required if we want optimizer to be relative to "baseline IFA" as we currently have it